source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -5948,6 +5825,10 @@ export class Container extends ContainerBase {
                 }
             }
         }
+        else if (isSingletonContainer(this) && this.singletonChild) {
```

Comment:
`this.singletonChild` is already checked by `isSingletonContainer()`.

Reply:
not quite -- `isSingletonContainer()` checks to see if `singletonChild` is either a valid `CardElement` OR is `undefined`